### PR TITLE
[12.0] [MIG] sale_order_type

### PR DESCRIPTION
**Pull request by angelmoya**

### CODE REVIEW

File: sale_order_type/README.rst [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--workflow-lightgray.png?logo=github
+    :target: https://github.com/OCA/sale-workflow/tree/11.0/sale_order_type
```
Comment (nikul-serpentcs) [inline sale_order_type/README.rst]: @angelmoya `target` 12.0

File: sale_order_type/README.rst [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    :alt: OCA/sale-workflow
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/sale-workflow-11-0/sale-workflow-11-0-sale_order_type
```
Comment (nikul-serpentcs) [inline sale_order_type/README.rst]: same here

File: sale_order_type/README.rst [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/167/11.0
```
Comment (nikul-serpentcs) [inline sale_order_type/README.rst]: same here

File: sale_order_type/README.rst [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+#. You can also define a type for a particular partner if you go to *Sales &
+   Purchases* and set a sale order type.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/sale-workflow/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/sale-workflow/issues/new?body=module:%20sale_order_type%0Aversion:%2011.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (nikul-serpentcs) [inline sale_order_type/README.rst]: @angelmoya same here, Change UR

File: sale_order_type/README.rst [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/sale-workflow <https://github.com/OCA/sale-workflow/tree/11.0/sale_order_type>`_ project on GitHub.
```
Comment (nikul-serpentcs) [inline sale_order_type/README.rst]: same here

File: sale_order_type/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+# Copyright 2015-2016 Oihane Crucelaegui <oihane@avanzosc.com>
+# Copyright 2015-2018 Pedro M. Baeza <pedro.baeza@tecnativa.com>
+# Copyright 2016 Lorenzo Battistini
+# Copyright 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# Copyright 2018 David Vidal <david.vidal@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Order Type",
+    "version": "11.0.1.3.0",
```
Comment (nikul-serpentcs) [inline sale_order_type/__manifest__.py]: "12.0.1.3.0",

File: sale_order_type/tests/test_sale_order_type.py [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+    def test_sale_order_confirm(self):
+        sale_type = self.sale_type
+        order_vals = self.get_sale_order_vals()
+        order = self.sale_order_model.create(order_vals)
+        order.onchange_partner_id()
+        self.assertTrue(order.type_id == sale_type)
+
+        order.onchange_type_id()
+        self.assertTrue(order.warehouse_id == sale_type.warehouse_id)
```
Comment (nikul-serpentcs) [inline sale_order_type/tests/test_sale_order_type.py]: @angelmoya use `assertEqual`

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Comment by antoniorubiopesol** [comment]
We are evaluating the module. Today we start to upload code.

**Comment by angelmoya** [comment]
Hi @nikul-serpentcs , @antoniorubiopesol did the work.

**Comment by pedrobaeza** [comment]
Please squash together adjacent oca-git-bot commits.

**Comment by angelmoya** [comment]
OK @pedrobaeza , done

File: sale_order_type/__manifest__.py (hunk 6 above)
Comment (alexey-pelykh) [reply sale_order_type/__manifest__.py]: `12.0.1.0.0` to be correct

File: sale_order_type/__manifest__.py [hunk 8]
```diff
@@ -8,13 +8,14 @@
...
     "name": "Sale Order Type",
-    "version": "11.0.1.3.0",
+    "version": "12.0.1.0.0",
     "category": "Sales Management",
     "author": "Grupo Vermon,"
               "AvanzOSC,"
               "Tecnativa,"
               "Agile Business Group,"
               "Niboo,"
+              "Pesol,"
```
Comment (pedrobaeza) [inline sale_order_type/__manifest__.py]: This simple migration doesn't grant you co-authorship.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: sale_order_type/__manifest__.py (hunk 8 above)
Comment (angelmoya) [reply sale_order_type/__manifest__.py]: Sorry, I will update it

Comment (angelmoya) [reply sale_order_type/__manifest__.py]: OK, done @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@alexey-pelykh @Nikul-Chaudhary please update your reviews

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Pull request merged**